### DISCUSSION TITLE
No-Code Customtkinter GUI Designer

### DISCUSSION THREAD

**Discussion by Akascape** [TomSchimansky/CustomTkinter#2187] [Showcase]
I made this software called **CTkDesigner** which is a no-code, drag and drop customtkinter gui design software. You just have to click on the widget to spawn it on the working window, then move it with the mouse, change parameters and export it to a python app. After exporting, you can edit your CTk app easily like adding command and functions. CTkDesigner will simply save your time.
  
[![Screenshot](https://github.com/Akascape/CTkDesigner-Support/assets/89206401/6435f49b-f7d0-4cba-8190-73cd71d77ac3)](https://ko-fi.com/s/6fca1ae70f)

![Screenshot](https://github.com/TomSchimansky/CustomTkinter/assets/89206401/847a1e0b-39aa-475e-8546-6346c3c00a16)

![Screenshot](https://github.com/Akascape/CTkDesigner-Support/assets/89206401/fc4da9cd-09ba-4f64-adc1-3fbfc50a40f8)

![Screenshot](https://github.com/Akascape/CTkDesigner-Support/assets/89206401/e8f1ea5a-24be-42fc-8513-eedba1662caf)
 
<br> <p align='center'>
![Screengif](https://github.com/Akascape/CTkDesigner-Support/assets/89206401/a2efbd88-eebf-4d34-aa36-d8c046584c93)
</p>

![Screenshot](https://github.com/Akascape/CTkDesigner-Support/assets/89206401/2aacd119-c580-4840-baae-1b57a7a45845)

<br> <p align='center'> [<img src="https://img.shields.io/badge/Get-CTkDesigner-informational?&logo=python&logoColor=yellow&color=green" width="500">](https://ko-fi.com/s/6fca1ae70f)  </br>

<br> <p align='center'> [<img src="https://img.shields.io/badge/Download-Trial-informational?&logo=python&logoColor=yellow&color=white" width="300">](https://github.com/Akascape/CTkDesigner-Support/releases/download/trial-version/CTkDesigner-Trial.zip)  </br>

 **Features:** 
✅ DRAG & DROP 
✅ Move and place widgets with mouse 
✅ Adjust all widget parameters 
✅ Precise place method 
✅ 15 core CTk widgets + 15 custom widgets
✅ Create and preview themes 
✅ Export to .py 
✅ Save/Open created templates again 
✅ All required functionalities and shortcuts added 
✅ WYSIWYG: what you see is what you get 

**Download size: 30 mb** 

Packed with pyinstaller
Proper documentation given inside a .pdf user manual

 **Support Page:** [CTkDesigner-Support](https://github.com/Akascape/CTkDesigner-Support)

## Watch Tutorial
<br> [<img src="https://img.youtube.com/vi/bIWLkiYeWFg/0.jpg" width=60% height=60%>](https://youtu.be/bIWLkiYeWFg)

**Comment by dan-ie** [comment]
and it is still impossible to change the windows icon :D

**Comment by DimaTepliakov** [comment]
This is crazy! thank you for that!

**Comment by SarthakTools** [comment]
Please see my customtkinter modern spotify music player built with python here
https://github.com/SarthakTools/Spotify

**Comment by Sonic3Modder** [comment]
ooh man, Wish this woesnt a pay for

**Comment by Jonasilver137** [comment]
aww man, why did you have to make it proprietary software? Personally, I think you should open-source your code so people can improve on it and make it better.
